pkgs/servers/apache-kafka/default.nix

```diff
@@ -1,22 +1,35 @@
-{ lib, stdenv, fetchurl, jdk8_headless, jdk11_headless, makeWrapper, bash, coreutils, gnugrep, gnused, ps,
+{ lib, stdenv, fetchurl, jdk17_headless, jdk11_headless, makeWrapper, bash, coreutils, gnugrep, gnused, ps,
   majorVersion ? "1.0" }:
 
 let
-  jre8 = jdk8_headless;
   jre11 = jdk11_headless;
+  jre   = jdk17_headless;
```

Comment:
```suggestion
```
I don't think we need that double aliasing.

Reply:
ok, removed.